### ISSUE TITLE
Search form box on site not visible in Windows High Contrast Mode

### ISSUE THREAD

**Issue by bisd** [localgovdrupal/localgov_base#499]
The input box on the search form on all pages of the site has the borders turned off in 'web/themes/contrib/localgov_base/css/components/header.css':

`.lgd-region--search .form-text {
  width: 100%;
  border: var(--header-search-input-border);
}`

This makes the box invisible in Windows HCM.

The box should have a border like the other boxes for "Docs", "Github" and "Apply" alongside it.

`border: 1px solid transparent;`

@msayoung 

![high_contrast](https://github.com/localgovdrupal/localgov_base/assets/2633608/c1a3fa6e-2f97-4305-bee6-88fd350912e5)

**Comment by markconroy** [comment]
Hi @bisd 

I don't have a windows computer to test/fix this. Would you be able to create a PR for it?

**Comment by bisd** [comment]
Sure Mark I will have a go this afternoon.
    On Friday, 17 November 2023 at 10:54:30 GMT, Mark Conroy ***@***.***> wrote:  
 
 


Hi @bisd

I don't have a windows computer to test/fix this. Would you be able to create a PR for it?
